libvips/foreign/magick.c

```diff
@@ -88,14 +87,6 @@ magick_sniff(const unsigned char *bytes, size_t length)
 		(bytes[4] != 0 || bytes[5] != 0))
 		return "ICO";
 
-	if (length >= 5 &&
-		bytes[0] == 0 &&
-		bytes[1] == 1 &&
-		bytes[2] == 0 &&
-		bytes[3] == 0 &&
-		bytes[4] == 0)
-		return "TTF";
```

Comment:
This was removed as it would be covered by:
https://github.com/ImageMagick/ImageMagick/blob/9b109732da21e1604ec28d32926c176bffb9d3e2/coders/ttf.h#L20

Reply:
Ah,  we still need the TTF sniffer as the TGA one is too greedy. Commit 95cdc5be014d70b3b4a06aa0267f41894fb9e1d0 re-adds this.